server/app/views/components/QuestionSortOption.java

```diff
@@ -0,0 +1,65 @@
+package views.components;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+
+/** Represents a sorting option for a list of Questions. */
+public enum QuestionSortOption {
+  LAST_MODIFIED("lastmodified", Optional.empty(), Optional.of("Last modified")),
```

Comment:
Can we have inline comments i.e. `/* dataAttribute */` to make clear what each param is?

Reply:
Done